### ISSUE TITLE
Include of non-modular header inside framework OpenTok

### ISSUE THREAD

**Issue by steverob** [opentok/opentok-react-native#17]
I'm trying to compile a ReactNative project with the OpenTok pod on iPhone 6 and the build is failing with these issues:

![skitch 1](https://user-images.githubusercontent.com/1220480/38681093-8d947dd0-3e85-11e8-92d1-ce3524a18458.png)

Version -   `OpenTok (2.13.0)`
xCode  - `9.4`
ReactNative - `0.52`

Tried the solutions here - https://forums.developer.apple.com/thread/23554 but to no avail. :(

**Comment by msach22** [comment]
@SteveMcFarlin just to verify, have you followed the iOS instructions and added the OpenTok SDK via Cocoapods and manually add the OpenTok React Native iOS files into the project?

**Comment by steverob** [comment]
Yes all those steps have been completed :( 

--
Regards
Steve Robinson

**Comment by msach22** [comment]
Thanks @steverob. I have a couple more questions if you don't mind:

- Can you verify that you created a Bridging-Header for your app?
- In your Build Settings, what is the Swift Compiler Language set to?
- Are you using the `.xcodeworkspace` file? This is required because you are using CocoaPods for the iOS SDK.

**Comment by steverob** [comment]
Hey @msach22 THanks!!

1. Yes, Bridging Header has been created
2. Swift Version 3

From project file
```
				SWIFT_OBJC_BRIDGING_HEADER = "ShowdownLive-Bridging-Header.h";
				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
				SWIFT_VERSION = 3.0;
```

3. Yes I'm opening the `.xcodeworkspace`

Thanks

**Comment by msach22** [comment]
@steverob Thanks for this! Can you set the `SWIFT_VERSION` to 4.0? Also, could you share the Bridging Header contents?

**Comment by steverob** [comment]
@msach22  will try to update the `SWIFT_VERSION`.

Bridging Header:

```
//
//  Use this file to import your target's public headers that you would like to expose to Swift.
//
#import <React/RCTBridgeModule.h>
#import <React/RCTBridgeMethod.h>
#import <React/RCTViewManager.h>
#import <React/RCTEventEmitter.h>
```

**Comment by msach22** [comment]
@steverob can you please add the following to your `Bridging-Header` file?

`#import <OpenTok/OpenTok.h>`

This should expose the OpenTok iOS SDK to Swift. Please let me know if this fixes it.

**Comment by jonathanwmaddison** [comment]
@msach22 I had a similar issue that only occurred after not working on my project for a few days (which had been working flawlessly). When I loaded it today, xcode complained about this. Adding `#import <OpenTok/OpenTok.h>` to Bridging-Header solved this for me.

**Comment by steverob** [comment]
Cool. This should be documented 

--
Regards
Steve Robinson

**Comment by msach22** [comment]
@jonathanwmaddison @steverob Yes, I figured that would've been it. I'll put up a PR updating the library and add it to the 0.4.1 release. Thanks for pointing this out!

I'll keep this issue open until the release goes out so people have it for reference.

**Comment by msach22** [comment]
Released in 0.4.1
